python/amici/sbml_preprocessing.py

```diff
@@ -0,0 +1,104 @@
+import libsbml
+
+
+def flatten_sbml(sbml_doc: libsbml.SBMLDocument):
```

Comment:
`sbml_doc` is not used anywhere. Why not directly pass the model instead of calling `.getModel()` so often?